internal/ethapi/tracer.go

```diff
@@ -240,21 +232,20 @@ func NewJavascriptTracer(code string, timeout time.Duration) (*JavascriptTracer,
 		db:         db,
 		dbvalue:    db.toValue(vm),
 		err:        nil,
-		timeout:    timeout,
 	}, nil
 }
 
+// Stop terminates execution of any JavaScript
+func (jst *JavascriptTracer) Stop(err error) {
+	jst.err = err
+	jst.vm.Interrupt <- func() {
+		panic(err)
+	}
+}
```

Comment:
I wrote a test that demonstrates that calls to Stop() between VM steps will be raised when the VM resumes, and so removed the extra assignment to `err`.